src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -1720,6 +1720,32 @@ void UpdateMemoryCost(in UInt256 position, in UInt256 length)
                             stack.PushUInt256(in baseFee);
                             break;
                         }
+                    case Instruction.DATAHASH:
+                        {
+                            if (!spec.IsEip4844Enabled)
+                            {
+                                EndInstructionTraceError(EvmExceptionType.BadInstruction);
+                                return CallResult.InvalidInstructionException;
+                            }
+
+                            if (!UpdateGas(GasCostOf.VeryLow, ref gasAvailable))
```

Comment:
NP: Use `GasCostOf.Datahash` instead and add `GastCostOf.Datahash = VeryLow` instead.